src/js/edu-benefits/5490/config/form.js

```diff
@@ -145,39 +147,75 @@ const formConfig = {
           path: 'benefits/eligibility',
           initialData: {},
           uiSchema: {
+            'view:benefitsDisclaimerChild': {
+              'ui:description': benefitsDisclaimerChild,
+              'ui:options': {
+                hideIf: form => _.get('relationship', form) !== 'child'
+              }
+            },
+            'view:benefitsDisclaimerSpouse': {
+              'ui:description': benefitsDisclaimerSpouse,
+              'ui:options': {
+                hideIf: form => _.get('relationship', form) !== 'spouse'
+              }
+            },
             benefit: {
               'ui:widget': 'radio',
               'ui:title': 'Select the benefit that is the best match for you:',
               'ui:options': {
                 labels: benefitsLabels,
-                nestedContent: {
-                  chapter33: Chapter33Warning,
-                  chapter35: Chapter35Warning
+                updateSchema: (data, form, schema) => {
+                  const relationship = _.get('relationship', form);
+                  const nestedContent = {
+                    chapter33: benefitSelectionWarning('chapter33', relationship),
+                    chapter35: benefitSelectionWarning('chapter35', relationship),
+                  };
+                  const uiOptions = _.get('benefitSelection.uiSchema.benefit.ui:options', form);
+                  uiOptions.nestedContent = nestedContent;
+                  return schema;
                 }
-              }
+              },
             },
             'view:benefitsRelinquishedInfo': {
               'ui:description': 'While receiving DEA or FRY scholarship benefits you may not receive payments of Dependency and Indemnity Compensation (DIC) or Pension and you may not be claimed as a dependent in a Compensation claim. If you are unsure of this decision it is strongly encouraged you talk with a VA counselor.',
+              'ui:options': {
+                hideIf: form => _.get('relationship', form) !== 'child'
+              }
             },
+            benefitsRelinquishedDate: _.merge(date.uiSchema('Effective date'), {
+              'ui:options': {
+                hideIf: form => _.get('relationship', form) !== 'child'
+              }
+            }),
             'view:benefitsRelinquishedWarning': {
-              'ui:description': benefitsRelinquishedWarning
+              'ui:description': benefitsRelinquishedWarning,
+              'ui:options': {
+                hideIf: form => _.get('relationship', form) !== 'child'
+              }
             },
-            benefitsRelinquishedDate: date.uiSchema('Effective date')
           },
           schema: {
             type: 'object',
             required: ['benefit', 'benefitsRelinquishedDate'],
             properties: {
+              'view:benefitsDisclaimerChild': {
```

Comment:
They are a big chunk of text just before the radio button question:
![image](https://cloud.githubusercontent.com/assets/25439097/24529926/7c9afe2e-1563-11e7-8f37-b71b845b1eee.png)